syntax = "proto2";
package echo;
option cc_generic_services = true;
message EchoRequest
{
    required string request = 1;
};
     
message EchoResponse
{
    required string response = 1;
};
     
service EchoService
{
    rpc Echo(EchoRequest) returns (EchoResponse);
    rpc Echo2(EchoRequest) returns (EchoResponse);
    rpc Echo3(EchoRequest) returns (EchoResponse);
};

service NoEchoService
{
	rpc noEcho(EchoRequest) returns (EchoResponse);
}
